docs/howto/logging.txt

```diff
@@ -0,0 +1,337 @@
+.. _logging-how-to:
+
+================================
+How to configure and use logging
```

Comment:
More an observation to consider than a suggestion per se, but all bar one of the other how-tos have their titles with the verb in present participle form. e.g. "Configuring and using logging".

Perhaps insisting on consistency here is overly prescriptive? Or maybe consistency is worth aiming for, but "How to..." is a better system to use than the current one?

Reply:
@tim-mccurrach Yes, you're one step ahead of me. A how-to guide should indicate in its title just what you will be shown how to do, so you can see whether your needs are likely to be answered in it.